fe1-web/src/features/home/interface/Configuration.ts

```diff
@@ -46,6 +53,11 @@ export interface HomeCompositionConfiguration {
    */
   useLaoList: () => HomeFeature.Lao[];
 
+  /**
+   * Gets the function to disconnect from the current lao
+   */
+  useDisconnectFromLao: () => () => void;
```

Comment:
would it be practical to disconnect by id , since we could be connected to multiple LAOs (in the future) ?

Reply:
Hmm I guess this is possible but I am not sure if it makes a lot of since at the moment. What would that function do? Read the currentId from the lao reducer, compare it to the passed one and only disconnect if it matches?

Also what id would we pass to the function? Just the currentId? Then it would seem a bit weird to me to start introducing partial support for this feature while the remaining codebase has no hints at that connections to multiple LAOs will ever be supported

Reply:
Fair enough :)